### DISCUSSION TITLE
PS2 Perk-A-Cola Jingles

### DISCUSSION THREAD

**Discussion by PHDFLOPPERGUY56** [nzp-team/nzportable#751] [Other Content Releases]
PS2 styled Perk-A-Cola & Pack-A-Punch jingle stings made by me in FL Studio using KiloHearts Bitcrusher
[sounds.zip](https://github.com/nzp-team/nzportable/files/15477742/sounds.zip)

**Comment by G00berF4n** [comment]
This is pretty neat, I like it :D.

**Comment by FilthyCasuall** [comment]
Beautiful
